### ISSUE TITLE
Добавить второй Java репозиторий

### ISSUE THREAD

**Issue by ValeraShimchuck** [sp-worlds/api-docs#11]
https://github.com/ValeraShimchuck/JSP
Почему? Потому что тут на русском, нормальная архитектура и имеет maven репозиторий из которого можно импортировать в разные системы сборки этот код. Также еще некоторые плюшки.

**Comment by teleport2** [comment]
Согласен, архитектура лучше, в отличие от моего. И maven репозиторий намного удобней импорта jar . Правда не понял при чем тут java 8?

**Comment by ValeraShimchuck** [comment]
Версия java - 1.8/8. Для того чтобы заимпортить твой api нужна 17 джава, а мой - 8.

**Comment by ronanru** [comment]
Добавь метод на подтверждение вебхука. Я не хочу добавлять библиотеку, если она не на 100% покрывает возможности api.

**Comment by ValeraShimchuck** [comment]
https://github.com/ValeraShimchuck/JSP/blob/master/src/main/java/ua/valeriishymchuk/jsp/validator/SHA256Validator.java вот он

**Comment by ronanru** [comment]
Так добавь этот метод в ReadMe, не понятно что он есть иначе

**Comment by ValeraShimchuck** [comment]
Обновил api и ReadMe. Ронан прийди.
